tests/appium/ios-ssh.e2e.js

```
@@ -0,0 +1,220 @@
+
+if (!shouldRun) {
+  console.log('skipped - set RUN_APPIUM_SSH_E2E=1 to run the Appium/XCUITest SSH scenario');
+  process.exit(0);
+}
+
+const APPIUM_SERVER_URL = process.env.APPIUM_SERVER_URL || 'http://127.0.0.1:4723';
+const IOS_APP = requiredEnv('IOS_APP');
+const SSH_PASSWORD = requiredEnv('SSH_PASSWORD');
+
+const sshOptions = {
+  address: process.env.SSH_HOST || 'localhost',
```

ESLint

[error] 'process' is not defined.

(no-undef)